lib/webauthn/relying_party.rb

```diff
@@ -31,6 +32,7 @@ def initialize(
       @algorithms = algorithms
       @encoding = encoding
       @origin = origin
+      @allowed_origins = allowed_origins.nil? ? [origin] : allowed_origins
```

Comment:
Is there any reason for setting `allowed_origins` to `origin` if the former is `nil`? It feels to me that I'd leave it `nil` for transparency if there's no need to set it 🤔